views/cases/family.hbs

```diff
@@ -1,54 +1,228 @@
-<div class='row justify-content-center' style="margin-top: 10px;">
-    <h3> Family </h3>
+<div class='row justify-content-center' style="margin-top: 15px;">
+    <h2> Family </h2>
 </div>
 
-<div class='row justify-content-center'>
-    <p>[Family Name]</p>
+<div class="card body p-2 text-center" style="border-radius: 1rem; margin-bottom: 15px;">
+    <div class='row justify-content-center' style="margin-top: 10px;">
+        <p style="margin:15px;">Case: Last Name</p>
+        <p style="margin:15px;">DYFS: 00000</p>
+        <p style="margin:15px;">Status: Closed</p>
+    </div>
 </div>
 
-<div class="row justify-content-center">
-
-    {{!-- <table class="table table-hover" style="background-color: silver; width:200%;"> --}}
-    {{#if familyMembers}}
-    <table class="table" style="width:70%;">
-        <tr>
-            <th>Name</th>
-            <td>DOB</td>
-            <td>Sex</td>
-            <td>Relationship</td>
-        </tr>
-        {{#each familyMembers}}
-            <tr>
-                {{#if this.name}}
-                    <th>{{this.name}}</th>
-                {{else}}
-                    <th>N/A</th>
-                {{/if}}
-                {{#if this.dob}}
-                    <td>{{this.dob}}</td>
-                {{else}}
-                    <th>N/A</th>
-                {{/if}}
-                {{#if this.sex}}
-                    <td>{{this.sex}}</td>
-                {{else}}
-                    <td>N/A</td>
-                {{/if}}
-                {{#if this.relationship}}
-                    <td>{{this.relationship}}</td>
-                {{else}}
-                    <td>N/A</td>
-                {{/if}}
-            </tr>
-        {{/each}}
-    </table>
-    {{else}}
-        <p>There are no recorded family members.</p>
-    {{/if}}
+<div class="card body p-2 text-center" style="border-radius: 1rem; margin-bottom: 15px;">
 
+    <button type="button" class="btn btn-link">Return to Family List</button>
+
+    <div class='row justify-content-center'>
+        <button style="margin:5px;" type="button" class="btn btn-secondary">Make Family Member A Patient</button>
+        <button style="margin:5px;" type="button" class="btn btn-secondary">Make Family Member An Actor</button>
+    </div>
+
+    <div id="btn-login" class="row justify-content-center">
+        <button type="submit" class="btn btn-primary btn-md">Update</button>
+    </div>
 </div>
 
+<div class="card body p-2 text-center" style="border-radius: 1rem; margin-bottom: 15px">
+    <h4> Family Members</h4>
+
+    <div class="form-group row">
+        <label style="margin:15px;" for="patient">Pre-fill form with patient info:</label>
+        <select name="patient" id="patient">
+            <option value="-Select-">-Select-</option>
```

Comment:
Not really, I just kept it as a place holder for the dropdown menu. Should I modify it to one or two people then or remove the drop down option?

Reply:
I guess for the time being you can get rid of the drop down option until we determine a way we want to GET patient's data that's already been entered into the system, not too sure but I do think this will be a route we should go in the future.  Besides that looks good

Reply:
Ok, I can remove the dropdown and make it a text box for now. Thanks for letting me know!

Reply:
@ashnarazdan123 Actually, you can leave this - guess it doesn't hurt to keep it since you took the time to make it, and given that most things on the site aren't fully functional at this state, it doesn't rly matter